tasks/fix-uid-gid-conflicts.yml

```diff
@@ -0,0 +1,71 @@
+---
+# this fixed UID/GID conflicts on new machine after migration
+
+# Backend: create copr user/group directly
+- block:
+  - name: Create copr group
+    group:
+      name: copr
+      gid: 986
+
+  - name: Create copr user
+    user:
+      name: copr
+      group: copr
+      uid: 989
+  
+  when: server_id == 'backend'
+
+# Keygen: move dhcpcd, then create copr-signer
+- name: Get user info for dhcpcd
+  getent:
+    database: passwd
+    key: dhcpcd
+  register: dhcpcd_user
+  failed_when: false
+  when: server_id == 'keygen'
+
+- block:
+  - name: Find files owned by UID 992
+    shell: find / -xdev -uid 992 2>/dev/null || true
+    register: files_uid_992
+    changed_when: false
+
+  - name: Change UID for user dhcpcd from 992 to 60992
+    user:
+      name: dhcpcd
+      uid: 60992
```

Comment:
output of this should be registered, and the subsequent chown should only ever be run when this successes